nuitka/nodes/ConstantRefNodes.py

```diff
@@ -357,6 +357,65 @@ def computeExpressionIter1(self, iter_node, trace_collection):
 
         return iter_node, None, None
 
+    def computeExpressionAny(self, any_node, trace_collection):
+        if not isIterableConstant(self.constant):
+            # An exception may be raised.
+            trace_collection.onExceptionRaiseExit(TypeError)
+
+            return getComputationResult(
+                node=any_node,
+                computation=lambda: any_node.simulator(self.constant),
+                description="Iteration of non-iterable constant.",
+            )
+
+        iteration_length = self.getIterationLength()
+
+        if (
+            iteration_length is not None
+            and iteration_length < 256
+            and self.canPredictIterationValues()
+        ):
+            all_false = True
+            for i in range(iteration_length):
+                truth_value = self.getIterationValue(i).getTruthValue()
```

Comment:
@kayhayen I just found out that this `self` doesn't have any method named `getIterationValue()` and we got away with it because we always tested it on `iteration_length`>256. However the `self` in `ConstantRefNodes.computeExpressionAny` has such a method which actually computes the result for us. So, what do you want me to do about it?

Reply:
Yes, I was bad there. You got to have a function indicating that ability, and then have an interface which is not in the general interface.

I think if you heed my request, these kinds of interfaces move into the new kind of handle, and are its methods rather. So, the function to return that handle would in base class return `None` meaning "I don't know", for some  a handle that says so, could be a shared re-used global variable, and from some an actual handle that can give iteration values, etc.

I sometimes do this on purpose, so if I use a method which I am not supposed to, it will crash. That is normally a sign of "still in flux". :-)